ext/readline/readline.stub.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/** @return string|false */
+function readline(string $prompt = null) {}
+
+/** @return mixed */
+function readline_info(string $varname = null, string $newvalue = null) {}
```

Comment:
The default values are not `null` (ZPP format string would have to be `"|s!z!"`); actually there are no default values (basically, readline_info() is overloaded for zero/one/two arguments). Use `UNKNOWN` instead.